### PR TITLE
fix: secret service macos bug

### PR DESCRIPTION
**Pull request by busla**

### CODE REVIEW

File: scripts/create-secret.sh [hunk 1]
```diff
@@ -1,100 +1,87 @@
 #!/bin/bash
-set -euo pipefail
+set -e
 
-#---------------------------------------------------------------------------------------------------------------#
-#--- Script for creating secrets (SecureString) in the AWS parameter store using the path /k8s/[secret-name]----#
-#---------------------------------------------------------------------------------------------------------------#
+export AWS_PAGER=""
```
Comment (sindrig) [inline scripts/create-secret.sh:4]: unused?

File: scripts/create-secret.sh [hunk 2]
```diff
@@ -104,69 +91,74 @@ function validate_length() {
...
+
+  if [ -n "$TAGS" ]
+  then
+    CMD="$CMD --tags $TAGS"
+  fi
+  eval "$CMD"
+  printf "%sDone!%s" "$GREEN" "$RESET"
 }
 
+case $1 in
```
Comment (sindrig) [inline scripts/create-secret.sh:156]: I don't get this case. Is this for testing purposes? Maybe add a comment about it then?

File: scripts/create-secret.sh (hunk 1 above)
Comment (busla) [reply scripts/create-secret.sh:4]: yes, disable the output from going to the pager

**Review by pshomov — APPROVED** [review]
LGTM I think

File: scripts/create-secret.sh (hunk 2 above)
Comment (busla) [reply scripts/create-secret.sh:156]: used in [shellspec](https://shellspec.info/) but threw shellspec out. Forgot to clean up this line.

File: scripts/create-secret.sh (hunk 1 above)
Comment (busla) [reply scripts/create-secret.sh:4]: so, no... its used 😄

**Review by sindrig — APPROVED** [review]

File: scripts/create-secret.sh (hunk 1 above)
Comment (sindrig) [reply scripts/create-secret.sh:4]: yep got it :)

**Pull request merged**